arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -1272,6 +1274,22 @@ Result RocksDBCollection::removeDocument(arangodb::transaction::Methods* trx,
 
   // disable indexing in this transaction if we are allowed to
   IndexingDisabler disabler(mthds, trx->isSingleOperationTransaction());
+  
```

Comment:
shouldn't we check whether this is actually the error first ?

Reply:
Done